flink-connector-postgres-cdc/src/test/java/com/ververica/cdc/connectors/postgres/PostgresTestBase.java

```diff
@@ -83,17 +85,28 @@ public static void startContainers() {
         LOG.info("Containers are started.");
     }
 
+    @AfterClass
+    public static void stopContainers() {
+        LOG.info("Stopping containers...");
+        POSTGRES_CONTAINER.stop();
+        LOG.info("Containers are stopped.");
+    }
+
     protected Connection getJdbcConnection(PostgreSQLContainer container) throws SQLException {
         return DriverManager.getConnection(
                 container.getJdbcUrl(), container.getUsername(), container.getPassword());
     }
 
     public static Connection getJdbcConnection(PostgreSQLContainer container, String databaseName)
             throws SQLException {
+        String jdbcUrl =
+                String.format(
+                        PostgresConnectionPoolFactory.JDBC_URL_PATTERN,
+                        container.getHost(),
+                        container.getMappedPort(PostgreSQLContainer.POSTGRESQL_PORT),
+                        databaseName);
```

Comment:
This change is the same as the original code. When will we get the different urls between the two code?

Reply:
The original code is  `container.withDatabaseName(databaseName).getJdbcUrl()`. It seems no difference, however, `withDatabaseName ` method will change the database name of POSTGRES_CONTAINER from 'postgres' to 'postgres-xxx'. As a static variable，POSTGRES_CONTAINER will influence all of later test units. They will start a database named 'postgres-xxx' rather than 'postgres'.

This change only get the jdbc url, don't change anything in container.